src/components/dropdown/dropdown.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Dropdown, DropdownOption } from './dropdown';
+import { Icon } from '../../new-components/icon/icon';
+
+<Meta
+  title="Components/Dropdown"
+  component={{ Dropdown }}/>
+
+# Dropdown
+This component is used to build more flexible dropdowns. It is based on the [Listbox of Reach UI](https://reach.tech/listbox/).
+
+<Canvas withSource="open">
+  <Story name="Regular">
+    <Dropdown defaultValue='default' arrow={<Icon show={'chevron_down'}/>}>
+        <DropdownOption popoverLabel={"Choose an option"} value='default' disabled>Choose an option</DropdownOption>
+        <DropdownOption popoverLabel={"Option 1"} value='option1'>1</DropdownOption>
+        <DropdownOption popoverLabel={"Option 2"} value='option2'>2</DropdownOption>
+        <DropdownOption popoverLabel={"Option 3"} value='option3'>3</DropdownOption>
```

Comment:
I think `popoverLabel` is your naming? It feels not good almost counter intuitive. It's the value displayed in the select when not opened right? I think we can do better. Can you propose a few names?

Reply:
I now understand how to use the ReachUI props and have imported them. Here the prop is simply called `label`. Is there a way in TS to rename a property name of an interface that you extend?